### PR TITLE
Add category and product config emod 49/107/108

### LINKED ISSUE #13: Confirmation button for reset, uninstall or disable modules is not correctly displayed
Module version 1.0.10

capture d'écran_1674

Module version 1.0.9

capture d'écran_1675

### PR DESCRIPTION
**Pull request by zalexki**
should also fix #13

### CODE REVIEW

File: views/js/controllers/configuration/back.js [hunk 1]
```diff
@@ -26,97 +26,43 @@
...
-    } else {
-        if ($(window).innerWidth() > 991) {
-            $('html, body').animate({scrollTop: $('.js-img-'+module).offset().top-topOffset}, 1000);
-        }
-    }
+    // TODO: dirty fix
+    // wait for new position of active
+    setTimeout(() => {
+        $('html, body').animate({scrollTop: $('.configuration-rectangle.active').position().top}, 1000);
+    }, 300);
```
Comment (Quetzacoalt91) [inline views/js/controllers/configuration/back.js]: No event can be caught instead? How can we be sure that 300ms is enough for environments?

File: ps_themecusto.php [hunk 2]
```diff
@@ -145,33 +148,34 @@ public function getTabNameByLangISO()
     {
         return array(
             $this->controller_name[1] => array(
-                'fr'    => 'Configuration page d\'accueil',
-                'en'    => 'Homepage Configuration',
-                'es'    => 'Configuración página de inicio',
-                'it'    => 'Configurazione homepage',
+                'fr' => 'Pages configuration',
+                'en' => 'Configuration pages',
```
[the remark this answers is not in the document]
Comment (zalexki) [reply ps_themecusto.php]: how so ?

File: views/js/controllers/configuration/back.js (hunk 1 above)
Comment (zalexki) [reply views/js/controllers/configuration/back.js]: ye there is a `TODO: fix dirty fix` just above :D i've modified with a more graceful way

File: ps_themecusto.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (Quetzacoalt91) [reply ps_themecusto.php]:
```suggestion
                'en' => 'Pages configuration',
```

[the remark this answers is not in the document]
Comment (zalexki) [reply ps_themecusto.php]: yeah you're right, i updated

File: .travis.yml [hunk 3]
```diff
@@ -23,3 +23,28 @@ script:
...
+  - rm -f .php_cs.*
+  - rm -f .travis.yml
+  - rm -f composer.*
+  - rm -f .gitignore
+  - rm -rf .git
+  - rm -rf tests
+  - mkdir -p /tmp/build/ps_themecusto
+  - mv * /tmp/build/ps_themecusto
+  - pushd /tmp/build
+  - zip /tmp/build/ps_themecusto.zip -r ps_themecusto/
```
Comment (Quetzacoalt91) [inline .travis.yml]: Please check again the original module, we removed the `mv` command as it does not move the hidden files.

Comment (zalexki) [reply .travis.yml]: done

File: views/js/controllers/configuration/back.js [hunk 4]
```diff
@@ -166,14 +167,22 @@ $(document).ready(function() {
         function() {
             let name = $(this).data('module_name');
             $('.module-list [data-module_name='+ name +']').addClass('active');
-            $(this).find('.on-element').removeClass('hide');
+            $(this).find('.on-element').addClass('hide');
```
Comment (Quetzacoalt91) [inline views/js/controllers/configuration/back.js]: What do you think about `.active-element` to properly distinguish the hovered from the active panel?

**Comment by sarahdib** [comment]
@zalexki 

When I disable/reset/uninstall a module the bloc disappear : 
https://drive.google.com/file/d/1K3Ffy-94RwzkA62OoXLq8OpVDNNfOWo0/view

Module customer review is missing in the list of module  available

**Comment by sarahdib** [comment]
@zalexki 
Module customer review is missing in the list of module available (product page -> product details) :)

**Comment by sarahdib** [comment]
@zalexki well done :) it's OK for QA

**Comment by Quetzacoalt91** [comment]
Thank you @zalexki

**Pull request merged**